### PR TITLE
portal-info: Add CommonID Field

### PR DESCRIPTION
**Pull request by 3v1n0**
Expose the application common ID in portal info, so that it can
be used to manage the application via Freedesktop Appplication API.

### CODE REVIEW

File: cmd/snap/cmd_routine_portal_info_test.go [hunk 1]
```diff
@@ -141,6 +147,67 @@ HasNetworkStatus=true
...
+							Name: "network-status",
+						},
+						Plug: client.PlugRef{
+							Snap: "hello",
+							Name: "network-status",
+						},
+						Interface: "network-status",
+					},
+				},
+			}
```
Comment (mardy) [inline cmd/snap/cmd_routine_portal_info_test.go:189]: Can we skip all this part about the network status? It does not seem relevant to the bit of code that you are testing :-)

Comment (3v1n0) [reply cmd/snap/cmd_routine_portal_info_test.go]: Yeah, done.

**Review by mvo5 — APPROVED** [review]
This looks fine, but I don't know too much about portals so a review from @jhenstridge would be ideal.

**Review by bboozzoo — APPROVED** [review]
LGTM, thanks!
